scim/resource_user.go

```diff
@@ -85,7 +85,7 @@ func createForceOverridesManuallyAddedUser(err error, d *schema.ResourceData, us
 	}
 	// corner-case for overriding manually provisioned users
 	userName := strings.ReplaceAll(u.UserName, "'", "")
-	force := fmt.Sprintf("User with username %s already exists.", userName)
+	force := fmt.Sprintf("User with email %s already exists in this account", userName)
```

Comment:
Will it continue to work with the workspace level identities?

Reply:
you are correct - updated the correct error message